website/source/docs/guides/managing-acl-policies.html.md

```diff
@@ -0,0 +1,150 @@
+``` 
+
+### Operations Example: Listing Consul Agents
+
+To view all Consul agents in the datacenter, you can use the `consul members` CLI command. You will need at least read privileges for every agent in the datacenter. Depending on your threat model you could have either of the following policies. 
+
+An individual rule and policy for each agent: this requires you to add a new policy to the token every time a new agent is added to the datacenter. Your token would have as many policies as agents in the datacenter. This method is ideal for a static environment, and for maximum security.
+
+```hcl
+agent "server-one" 
+  { policy = "read"
+  }
+```
+
+If you have a dynamic or very large environment, you may want to consider creating one policy that can apply to all agents, to reduce the Operation Team’s workload. 
+
+```hcl
+agent_prefix "" 
+  { policy = "read"
+  }
+```
+
+## Secure Access Control: Operator-Only Access
+
+The most secure access control implementation restricts tokens with `acl=”write”` policies to only one or a few trusted operators. Tokens with the policy `acl = "write"` grant the holder unlimited privileges, because they can generate tokens with any other resource and policy. The operators are responsible for creating all other policies and tokens that grant limited access to the datacenter. We refer to this implementation as the operator-only implementation. This implementation type is the most secure, and most complex to manage.  
+
+For this implementation type operators are responsible for managing policies and tokens for: 
+- service registration
+- Connect proxy registration
+- intention management
+- agent management
+- API, CLI, and UI access
+
+For static or non-containerized workflows, this implementation type is straightforward and the operator's workload scales linearly. If the workflow is dynamic, implementing an automation tool will ensure the operator's workload does not scale exponentially. 
+
+!> If you need to share token generation responsibilities with other teams, anyone with the ability to create tokens (`acl = "write"`) will also have unrestricted access to the datacenter. That individual will have the ability to deregister agents and services, and delete data from Consul KV. 
+
+### Operator-Only Implementation Example 
+
+In this following example Operators retain responsibility for service token management, but delegate access control between Connect-enabled services to the security team. 
+
+-> Note: The service registration examples describe the token generation process for per-service tokens, therefore, they are only applicable if you are able to create per-service tokens. 
+
+Initially, the Operator creates a single token with intention management privileges for the Security Team, and another service token for the Developer. For intention management the `intention` policy disposition should be included in the service rule.
```

Comment:
nit: You spoke about two tokens one for sec team one for dev but then didn't explicitly specify which one you mean in the second sentence. It's perfectly clear if you follow it since you said "intention management" was for "security" in the last sentence but I wonder if it could be even clearer if you removed the mention of a second token or made the second sentence more explicit?

For example:

> Initially, the Operator creates a single token with intention management privileges for the Security Team. Developers will be given separate tokens to enable them to register their applications without having access to manage Intentions.
>
> The security team's token must include the `intention` policy disposition in the service rule for any services they should be allowed to manage Intentions for.

Reply:
Updated!